src/axolotl/core/trainer_builder.py

```diff
@@ -1232,16 +1232,13 @@ def get_callbacks(self) -> List[TrainerCallback]:
                 SaveAxolotlConfigtoWandBCallback(self.cfg.axolotl_config_path)
             )
         if self.cfg.use_mlflow and is_mlflow_available():
-            from transformers.integrations.integration_utils import MLflowCallback
-
             from axolotl.utils.callbacks.mlflow_ import (
                 SaveAxolotlConfigtoMlflowCallback,
             )
 
             callbacks.extend(
                 [
                     SaveAxolotlConfigtoMlflowCallback(self.cfg.axolotl_config_path),
-                    MLflowCallback,
```

Comment:
this isn't really duplicate is it? One is our custom callback to save the axolotl YAML to mlflow, the other is the MLFlow callback that transformers uses to log to mlflow.

Reply:
We also have this which transformers internally may use to add the callback https://github.com/axolotl-ai-cloud/axolotl/blob/b620ed94d0682549433640ee03675ac9e2277cd6/src/axolotl/core/trainer_builder.py#L1587-L1588